plugins/packages/RPM/SPECS/plugin.spec

```diff
@@ -121,4 +121,7 @@ set_files_ownership () {
 echo "Setting ownership of FogLAMP files"
 set_files_ownership
 
+# Install any Python dependencies
+find /usr/local/foglamp/__INSTALL_DIR__ -name "requirements*.txt" -exec pip3 install -Ir "{}" --no-cache-dir \;
```

Comment:
As we packaged requirements file under `/usr/local/foglamp/python/` directory. So above line will never execute as we find the file in `/usr/local/foglamp/__INSTALL_DIR__`.
**For example**:
```
/usr/local/foglamp/python/foglamp/plugins/south/coap/README.rst
/usr/local/foglamp/python/foglamp/plugins/south/coap/__init__.py
/usr/local/foglamp/python/foglamp/plugins/south/coap/coap.py
/usr/local/foglamp/python/requirements-coap.txt
```
So first we need the name of plugin and then find requirements under `/usr/local/foglamp/python`. We may fix with below code OR any better solution

```
plugin_name=`echo __NAME__ | sed 's/.*-//'`

# Install any Python dependencies
find /usr/local/foglamp/python -name "requirements-${plugin_name}.txt" -exec pip3 install -Ir "{}" --no-cache-dir \;
```